### PR TITLE
draft of the 15 year blog post

### PR DESCRIPTION
**Pull request by cfbolz**
first go at a blog anniversary post

### CODE REVIEW

File: posts/2022/10/blog-15-years.rst [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+.. tags: meta
+.. category:
+.. link:
+.. description:
+.. type: rest
+.. author: Carl Friedrich Bolz-Tereick
+
+Exactly 15 years ago today we wrote the `first blog post on the PyPy blog`_!
+Over the years, we have written 423 posts, from the `shortest`_ to the
+`longest`_. In 2021 we moved_ from blogger_ to our own do domain.
```
Comment (mattip) [inline posts/2022/10/blog-15-years.rst:13]:
```suggestion
`longest`_. In 2021 we moved_ from blogger_ to our own domain.
```

**Review by mattip — COMMENTED** [review]
Fun, thanks. Found only one typo.

**Review by antocuni — COMMENTED** [review]
Nice blog post, thanks for writing :).
In terms of achievements, maybe we can also mention that at least one of our post was on top of hackernews:
https://news.ycombinator.com/front?day=2018-09-21
https://www.pypy.org/posts/2018/09/inside-cpyext-why-emulating-cpython-c-8083064623681286567.html

Moreover, I'm particularly fond of that blog post because it's basically a manifesto of why HPy is a good idea :).

**Comment by antocuni** [comment]
Also, I don't know if you want to include it in the blog post, but I got curious about who wrote the most posts :)
I think the following command answers the question, but I'm not sure it's 100% correct of course. I had to exclude hidden files else it would count also comments:
```
$ grep -h '\.\. authors' `find . -not -path '*/.*' -type f` | sort | uniq -c | sort -nr
    103 .. authors: Maciej Fijalkowski
     94 .. authors: Carl Friedrich Bolz-Tereick
     71 .. authors: Armin Rigo
     38 .. authors: Antonio Cuni
     35 .. authors: mattip
     11 .. authors: Philip Jenvey
     10 .. authors: Alex
      8 .. authors: Unknown
      7 .. authors: holger krekel
      6 .. authors: Romain Guillebert
      5 .. authors: David Schneider
      4 .. authors: Brian Kearns
      2 .. authors: Wim Lavrijsen
      2 .. authors: Samuele Pedroni
      2 .. authors: hodgestar
      2 .. authors: Hakan Ardo
      2 .. authors: Alexander Schremmer
      1 .. authors: Michael Foord
      1 .. authors: Benjamin Peterson
```

**Comment by mattip** [comment]
Drop the `s` from `authors`:
```
 grep -h '\.\. author' `find . -not -path '*/.*' -type f` | sort | uniq -c | sort -nr
    103 .. authors: Maciej Fijalkowski
     94 .. authors: Carl Friedrich Bolz-Tereick
     71 .. authors: Armin Rigo
     38 .. authors: Antonio Cuni
     35 .. authors: mattip
     11 .. authors: Philip Jenvey
     10 .. authors: Alex
      9 .. author: The Nikola Team
      8 .. author: The PyPy Team
      8 .. authors: Unknown
      7 .. authors: holger krekel
      6 .. authors: Romain Guillebert
      6 .. author: Carl Friedrich Bolz-Tereick
      5 .. authors: David Schneider
      4 .. authors: Brian Kearns
      2 .. authors: Wim Lavrijsen
      2 .. authors: Samuele Pedroni
      2 .. authors: hodgestar
      2 .. authors: Hakan Ardo
      2 .. authors: Alexander Schremmer
      2 .. author: mattip
      2 .. author: antocuni
      1 .. author: Vilhjálmur Þorsteinsson
      1     .. author_url: https://felix.fontein.de
      1 .. authors: Michael Foord
      1 .. authors: Benjamin Peterson
      1 .. author: Roberto Alsina
      1     .. author_IP: 1.2.3.4
      1     .. author: felix
      1     .. author_email: felix@fontein.de
      1 .. author: docutils contributors
```

**Comment by antocuni** [comment]
I wrote a quicky&dirty script to scrape HN and get the articles which were on its front page on the day they were published:
https://github.com/pypy/pypy.org/blob/5d221d778a7ff0fb7ac0892d15b5aa952d9922f7/posts/2022/10/get_hackernews_stats.py

Here is the result:
```
Rank Points       Date
---- ------ ----------
   1    242 2014-07-05    PyPy-STM: first “interesting” release (morepypy.blogspot.com)    http://morepypy.blogspot.com/2014/07/pypy-stm-first-interesting-release.html
   1    644 2017-08-14    Let's Remove the Global Interpreter Lock (morepypy.blogspot.com)    https://morepypy.blogspot.com/2017/08/lets-remove-global-interpreter-lock.html
   1    173 2018-09-21    Inside cpyext: Why emulating CPython C API is so Hard (morepypy.blogspot.com)    https://morepypy.blogspot.com/2018/09/inside-cpyext-why-emulating-cpython-c.html
   4    179 2011-01-21    Pypy wants you - now's the time to start contributing to Pypy (morepypy.blogspot.com)    http://morepypy.blogspot.com/2011/01/pypy-wants-you.html?uhttp://morepypy.blogspot.com/2011/01/pypy-wants-you.html
   4    299 2014-06-20    The first stable release of PyPy3 (morepypy.blogspot.com)    http://morepypy.blogspot.com/2014/06/pypy3-231-fulcrum.html
   5    233 2011-08-02    PyPy is faster than C, again: string formatting (morepypy.blogspot.com)    http://morepypy.blogspot.com/2011/08/pypy-is-faster-than-c-again-string.html
   6    176 2012-11-22    PyPy 2.0 beta 1 released (morepypy.blogspot.se)    http://morepypy.blogspot.se/2012/11/pypy-20-beta-1.html
   6    226 2019-01-03    PyPy for low-latency systems (morepypy.blogspot.com)    https://morepypy.blogspot.com/2019/01/pypy-for-low-latency-systems.html
   7    166 2015-09-09    PyPy warmup improvements (morepypy.blogspot.com)    http://morepypy.blogspot.com/2015/09/pypy-warmup-improvements.html
   8    165 2011-06-29    Global Interpreter Lock, or how to kill it (morepypy.blogspot.com)    http://morepypy.blogspot.com/2011/06/global-interpreter-lock-or-how-to-kill.html
   8    280 2013-05-09    PyPy 2.0 Released (morepypy.blogspot.com)    http://morepypy.blogspot.com/2013/05/pypy-20-einstein-sandwich.html
   9    244 2011-08-18    PyPy 1.6 Released - Full Python 2.7.1 Implementation (morepypy.blogspot.com)    http://morepypy.blogspot.com/2011/08/pypy-16-kickass-panda.html
  10    160 2014-11-17    Tornado Without a GIL on PyPy STM (morepypy.blogspot.com)    http://morepypy.blogspot.com/2014/11/tornado-without-gil-on-pypy-stm.html
  10    169 2016-07-08    Reverse Debugging for Python (morepypy.blogspot.com)    https://morepypy.blogspot.com/2016/07/reverse-debugging-for-python.html
  11    178 2012-01-27    PyPy funded to begin support for Python 3 and Numpy (morepypy.blogspot.com)    http://morepypy.blogspot.com/2012/01/py3k-and-numpy-first-stage-thanks-to.html
  13    167 2012-02-10    PyPy 1.8 - business as usual (morepypy.blogspot.com)    http://morepypy.blogspot.com/2012/02/pypy-18-business-as-usual.html
  14    200 2013-02-28    Ten years of PyPy (morepypy.blogspot.de)    http://morepypy.blogspot.de/2013/02/10-years-of-pypy.html
  14    192 2019-10-08    PyPy's New JSON Parser (morepypy.blogspot.com)    https://morepypy.blogspot.com/2019/10/pypys-new-json-parser.html
  17    262 2020-12-31    PyPy Project looking for sponsorship to add support for Apple Silicon (morepypy.blogspot.com)    https://morepypy.blogspot.com/2020/12/mac-meets-arm64.html
  18    183 2018-09-09    The First 15 Years of PyPy – A Personal Retrospective (morepypy.blogspot.com)    https://morepypy.blogspot.com/2018/09/the-first-15-years-of-pypy.html
  21    303 2015-10-29    PyPy 4.0.0 Released – A Jit with SIMD Vectorization and More (morepypy.blogspot.com)    http://morepypy.blogspot.com/2015/10/pypy-400-released-jit-with-simd.html
  30    284 2016-03-10    PyPy 5.0 Released (morepypy.blogspot.com)    http://morepypy.blogspot.com/2016/03/pypy-50-released.html
  30    316 2016-08-09    PyPy gets funding from Mozilla for Python 3.5 support (morepypy.blogspot.com)    https://morepypy.blogspot.com/2016/08/pypy-gets-funding-from-mozilla-for.html
  30    221 2016-10-12    PyPy3 5.5.0 released (morepypy.blogspot.com)    https://morepypy.blogspot.com/2016/10/pypy3-550-released.html
  30    180 2017-10-05    PyPy v5.9 Released (morepypy.blogspot.com)    https://morepypy.blogspot.com/2017/10/pypy-v59-released-now-supports-pandas.html
```

**Pull request merged**